src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java

```diff
@@ -792,136 +794,136 @@ private AdsDataProvider() {
+//                .put("slotSize", new Dimension(300, 250))
+//                .put("lineItemId", 269679732)
+//                .put("src", "gpt")
+//                .build()
+//        },
+//        {
+//            new Page("project43", "SyntheticTests/Slots/Skyscrapers/1x300x250,1x300x600" + INSTANT_GLOBAL_ADMIX_SWITCHED_OFF),
+//            "",
+//            ImmutableMap.<String, Object>builder()
+//                .put("slotName", AdsContent.LEFT_SKYSCRAPPER_2)
+//                .put("slotSize", new Dimension(300, 250))
+//                .put("lineItemId", 260204412)
+//                .put("src", "gpt")
+//                .build()
+//        },
+//        {
+//            new Page("project43", "SyntheticTests/Slots/Skyscrapers/2x300x250" + INSTANT_GLOBAL_ADMIX_SWITCHED_OFF),
+//            "",
+//            ImmutableMap.<String, Object>builder()
+//                .put("slotName", AdsContent.LEFT_SKYSCRAPPER_3)
+//                .put("slotSize", new Dimension(300, 250))
+//                .put("lineItemId", 260206692)
+//                .put("src", "gpt")
+//                .build()
+//        },
+//        {
+//            new Page("project43", "SyntheticTests/Oasis/FloatingMedrecOnLongPage/160x600" + INSTANT_GLOBAL_ADMIX_SWITCHED_OFF),
+//            "",
+//            ImmutableMap.<String, Object>builder()
+//                .put("slotName", AdsContent.FLOATING_MEDREC)
+//                .put("slotSize", new Dimension(160, 600))
+//                .put("lineItemId", 270616092)
+//                .put("src", "gpt")
+//                .build()
+//        },
+//        {
+//            new Page("project43", "SyntheticTests/Slots/Size/120x600" + INSTANT_GLOBAL_ADMIX_SWITCHED_OFF),
+//            "",
+//            ImmutableMap.<String, Object>builder()
+//                .put("slotName", AdsContent.LEFT_SKYSCRAPPER_2)
+//                .put("slotSize", new Dimension(120, 600))
+//                .put("lineItemId", 257673852)
+//                .put("src", "gpt")
+//                .build()
+//        },
+//        {
+//            new Page("project43", "SyntheticTests/Slots/Size/120x600" + INSTANT_GLOBAL_ADMIX_SWITCHED_OFF),
+//            "",
+//            ImmutableMap.<String, Object>builder()
+//                .put("slotName", AdsContent.FLOATING_MEDREC)
+//                .put("slotSize", new Dimension(120, 600))
+//                .put("lineItemId", 257673852)
+//                .put("src", "gpt")
+//                .build()
+//        },
+//        {
+//            new Page("project43", "SyntheticTests/Slots/Size/120x600" + INSTANT_GLOBAL_ADMIX_SWITCHED_OFF),
+//            "",
+//            ImmutableMap.<String, Object>builder()
+//                .put("slotName", AdsContent.LEFT_SKYSCRAPPER_3)
+//                .put("slotSize", new Dimension(120, 600))
+//                .put("lineItemId", 257673852)
+//                .put("src", "gpt")
+//                .build()
+//        },
+//        {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3ACommentedOutCodeLine)